core/src/main/java/kafka/server/share/SharePartitionManager.java

```diff
@@ -427,6 +427,26 @@ public List<TopicIdPartition> cachedTopicIdPartitionsInShareSession(String group
         return cachedTopicIdPartitions;
     }
 
+    /**
+     * The acknowledgeShareSessionCacheUpdate method is used to update the share session cache before acknowledgements are handled
+     * either as part of shareFetch request or shareAcknowledge request
+     * @param groupId The group id in the request.
+     * @param memberId The member id of the client in the request.
+     * @param reqEpoch The request epoch.
+     */
+    public void acknowledgeShareSessionCacheUpdate(String groupId, Uuid memberId, int reqEpoch) {
```

Comment:
Is this method being used anywhere now?

Reply:
nope, already taken care of in the latest commit. Thanks